script/validate-data/index.ts

```diff
@@ -49,7 +49,7 @@ async function checkWorkflows(folders: string[]): Promise<WorkflowWithErrors[]>
     });
 
     for (const e of dir) {
-      if (e.isFile()) {
+      if (e.isFile() && extname(e.name) === ".yml") {
```

Comment:
This looks good to me. Would you like to check for both `.yml` and `.yaml` here?

Reply:
👍 will do!